src/instructlab/model/backends/backends.py

```diff
@@ -111,6 +114,48 @@ def safe_close_all(resources: typing.Iterable[Closeable]):
             resource.close()
 
 
+def is_model_safetensors(model_path: pathlib.Path) -> bool:
+    """
+    Check if provided path to model represents directory containing a safetensors representation
+    of a model. Directory must contain a specific set of files to qualify as a safetensors model directory
+    Args:
+        model_path (Path): The path to the model directory
+    Returns:
+        bool: True if the model is a safetensors model, False otherwise.
+    """
+
+    try:
+        required_files = frozenset(
+            {
+                "config.json",
+                "*.safetensors",
+                "tokenizer.json",
+                "tokenizer.model",
+                "tokenizer_config.json",
+            }
+        )
```

Comment:
There is no need to create a frozen set here. The variable never leaves the scope of the function.